nailgun/entities.py

```diff
@@ -138,6 +138,31 @@ def _check_for_value(field_name, field_values):
         )
 
 
+def _get_org_attrs(server_config, organization_label):
```

Comment:
This method is not the first one in a list of such methods. This should only exist until the entity search mixin comes along and renders this redundant.

This method exists because there are several unique but highly similar bugs. Here's one of them:

> One can read a content view by issuing an HTTP GET request to `/katello/api/v2/products/:id`. This request returns JSON data in this form:
>
>     {
>         'id': …,
>         'organization' {'label': '…', 'name': '…'},
>         …
>     }
>
> Notice how the "organization" dict does not include an ID. This makes it impossible to know which organization a content view is associated with without performing a search against the server.